tests/drivers/adc/adc_accuracy_test/testcase.yaml

```diff
@@ -21,5 +21,5 @@ tests:
       - ek_ra8m1
       - frdm_mcxc242
       - nrf52840dk/nrf52840
-      - nrf54l15pdk/nrf54l15/cpuapp
+      - nrf54l15dk/nrf54l15/cpuapp
```

Comment:
Wrong commit, This should be corrected in [tests: adc_accuracy: add overlay for nrf54l15dk.](https://github.com/zephyrproject-rtos/zephyr/pull/76088/commits/c883b83b93e9c12f4a9e3fcd649ee81ef157ded0).